core/orch_test.go

```diff
@@ -955,13 +1171,29 @@ func TestProcessPayment_AcceptablePriceError_IncreasesCredit_ReturnsError(t *tes
 }
 
 func TestProcessPayment_UnacceptablePriceError_ReturnsError_DoesNotIncreaseCredit(t *testing.T) {
-	n, _ := NewLivepeerNode(nil, "", nil)
+	dbh, dbraw, err := common.TempDB(t)
+	require.Nil(t, err)
+	defer dbh.Close()
+	defer dbraw.Close()
+	n, _ := NewLivepeerNode(nil, "", dbh)
 	n.Balances = NewAddressBalances(5 * time.Second)
 	recipient := new(pm.MockRecipient)
 	n.Recipient = recipient
-	orch := NewOrchestrator(n)
+	rm := &stubRoundsManager{
+		round: big.NewInt(10),
+	}
+	orch := NewOrchestrator(n, rm)
+
+	orch.address = pm.RandAddress()
 	orch.node.SetBasePrice(big.NewRat(5, 1))
 	orch.node.ErrorMonitor = NewErrorMonitor(0, make(chan struct{}))
+
+	dbh.UpdateOrch(&common.DBOrch{
```

Comment:
Seems that we have to repeatedly initialize a DB with a single O in many of this PR's tests. Worth considering introducing a test helper for initializing a DB that starts off in the right state. Something like:

```
func tempDBWithOrch(t *testing.T, orch *common.DBOrch) (*common.DB, *sql.DB) {
    return tempDBWithOrchs(t, []*common.DBOrch{orch})
}

func tempDBWithOrchs(t *testing.T, orchs []*common.DBOrch) (*common.DB, *sql.DB) {
    dbh, dbraw, err := common.TempDB(t)
    require.Nil(t, err)

    for _, orch := range orchs {
        require.Nil(t, dbh.UpdateOrch(orch))
    }

    return dbh, dbraw
}
```

(The extra `tempDBWithOrchs()` might be overkill)

Then, the helper could be used to reduce the boilerplate in each updated test:

```
addr := pm.RandAddress()
dbh, dbraw := tempDBWithOrch(&common.DBOrch{...})
defer dbh.Close()
defer dbraw.Close()
```

WDYT?

Reply:
fixed in 35d8e94a